backend/services/client.js

```diff
@@ -115,3 +115,43 @@ exports.changePasswordService = async function({ token, newPassword }) {
 
   return {};
 };
+
+exports.forgetPasswordService = async function({email, type}) {
+  const clientWithEmail = await ClientDataAccess.getClientByEmailAndTypeDB(email, type);
+
+  if (isNull(clientWithEmail)) {
+    throw new AppError(Messages.RETURN_MESSAGES.ERR_CLIENT_DOES_NOT_EXIST);
+  }
+
+  const resetPasswordToken = Date.now() + sha1(newClient._id.toString() + Date.now());
+  const resetURL = `http://${Config.url}:${Config.port}/client/resetPassword?resetPasswordToken=${resetPasswordToken}`;
```

Comment:
```suggestion
  const resetURL = `http://${Config.hostAddr}:${Config.port}/client/resetPassword?resetPasswordToken=${resetPasswordToken}`;
```
I changed this part in my last commit. It is now outdated.

Reply:
@mtyazici Resolved.